src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3ResponseHelper.java

```diff
@@ -65,4 +70,84 @@ public static List<String> validateEmploymentDates(CaseData caseData) {
 
         return errors;
     }
+
+    /**
+     * Adds a group of documents to the ET3 response collection so that it can be displayed on the document tab.
+     * @param caseData data for the current case.
+     * @param documentTypeItemList the documents to be added to the document tab.
+     */
+    public static void addDocuments(CaseData caseData, List<DocumentTypeItem> documentTypeItemList) {
+        if (CollectionUtils.isEmpty(documentTypeItemList)) {
+            return;
+        }
+
+        if (CollectionUtils.isEmpty(caseData.getEt3ResponseDocumentCollection())) {
+            caseData.setEt3ResponseDocumentCollection(new ArrayList<>());
+        }
+
+        documentTypeItemList.stream()
+            .filter(d -> !documentExistsOnCollection(caseData.getEt3ResponseDocumentCollection(),
+                d.getValue().getUploadedDocument().getDocumentBinaryUrl()))
+            .forEach(documentToAdd -> saveFileToCollection(caseData, documentToAdd.getValue()));
+    }
+
+    /**
+     * Adds a document to the ET3 response collection so that it can be displayed on the document tab.
+     * @param caseData data for the current case.
+     * @param documentToAdd the document to be added to the document tab.
+     */
+    public static void addDocument(CaseData caseData, UploadedDocumentType documentToAdd) {
+        if (documentToAdd == null) {
+            return;
+        }
+
+        if (caseData.getEt3ResponseDocumentCollection() == null) {
+            caseData.setEt3ResponseDocumentCollection(new ArrayList<>());
+        } else if (
+            documentExistsOnCollection(caseData.getEt3ResponseDocumentCollection(),
```

Comment:
suggestion: extra out `caseData.getEt3ResponseDocumentCollection()` to a local variable, its used twice and would allow the else if to be on a single line